doc/source/dev/core-dev/distributing.rst.inc

```diff
@@ -106,11 +107,12 @@ and distributing them on PyPI or elsewhere.
 
 **General**
 
-- A binary is specific for a single Python version (because different Python
-  versions aren't ABI-compatible, at least up to Python 3.4).
-- Build against the lowest NumPy version that you need to support, then it will
-  work for all NumPy versions with the same major version number (NumPy does
-  maintain backwards ABI compatibility).
+- A binary is specific for a single (major) Python version (because different
+  major Python versions aren't ABI-compatible, at least up to Python 3.12).
+- Build against the lowest NumPy 2.x version that you need to support, then it
```

Comment:
I think this may be stricter than the reality, though I don't know if it is necessarily bad guidance in the 2.x series.

Reply:
I'd completely remove the "lowest" part, since it is no longer true and it's really robust to not do so.